saleor/page/migrations/0030_slug_translation_unique_constraint.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.postgres.indexes import BTreeIndex
+from django.contrib.postgres.operations import AddIndexConcurrently
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ("page", "0029_add_page_translation_slug"),
+    ]
+
+    operations = [
+        AddIndexConcurrently(
```

Comment:
This needed to be done slightly differently, an example from PR here: https://github.com/saleor/saleor/pull/15986/files#diff-e4e911ec8d5a8922cc7b73fb117b5e1f8755d015d33c17e0f80969f96079c869

The point is to make sure that postgres use the index we created (and unfortunately we need to write raw sql, as there is no support for uniq indexes creation within `AddIndexConcurrently`).

I know the docs are missing this, I will add it soon there as well.

Reply:
Small tip from me, there is very useful command that allows us to see sql's created from django:
`python manage.py sqlmigrate page 0030`